roles/openshift_examples/tasks/main.yml

```diff
@@ -36,6 +36,7 @@
   unarchive:
     src: "{{ copy_examples_mktemp.stdout }}/openshift-examples.tar"
     dest: "{{ examples_base }}/"
+    mode: 0755
```

Comment:
grrrrrrrrrrrrrrrrr. '0755' is getting applied to all directories and files under the `examples/` path. Files should be more like `0644` :confounded: